modules/creating-oci-resources-services.adoc

```diff
@@ -4,29 +4,18 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="creating-oci-resources-services_{context}"]
-= Creating OCI resources and services
+= Creating {oci} resources and services
 
 Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization's requirements.
 
 By creating a compartment, you can better organize, restrict access, and set usage limits to {oci} resources. Additionally, by creating an object storage bucket, you can safely and securely store the ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
 
 .Prerequisites
 
-* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/csgsg/create-users-and-assign-roles.html#GUID-1041AC78-9E43-42CF-B5A5-73F8B8FEB39A[Create Users and Assign Roles] in the Oracle documentation.
+* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm[Prerequisites] in the _Oracle_ documentation.
```

Comment:
It seems you do not italicize Oracle documentation more than you do. Should this be consistent?
```suggestion
* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm[Prerequisites] in the Oracle documentation.
```

Reply:
OK as per: 
![Screenshot from 2024-05-01 15-34-27](https://github.com/openshift/openshift-docs/assets/57954076/d20c4172-9c20-44b8-b9a4-aca4fbdc5d8e)